core/shared/src/main/scala/com/monovore/decline/CommandApp.scala

```diff
@@ -22,7 +22,7 @@ class CommandApp(command: Command[Unit]) {
   }
 
   def main(args: Array[String]): Unit =
-    command.parse(args) match {
+    command.parse(PlatformApp.ambientArgs getOrElse args) match {
```

Comment:
I'm confused about this one. This is just a method, and it requires `Array[String]` to call the function. It seems weird to me to discard what was passed after being called on a certain platform (I guess JS will have ambientArgs but not scala-jvm or scala-native).

Seems to me this should be higher up. So the caller of this method should call it with ambientArgs.